docs/developer/build.md

```diff
@@ -68,6 +66,7 @@ Connect your device to your machine over USB, and reboot your device into the bo
 
 ...and flash your build:
 
+    $ cout  # cd $OUT
```

Comment:
This is unnecessary since fastboot will automatically search `$ANDROID_PRODUCT_OUT` for images.

Reply:
redacted, thanks :+1: